src/tool_operate.c

```diff
@@ -1230,6 +1230,17 @@ static CURLcode single_transfer(struct GlobalConfig *global,
         }
 #endif
 
+#if defined(__unix) || defined (WIN32)
```

Comment:
I think this is an unnecessary precaution. If there's a platform with a problem with this, we should add a reversed check for that specific one.

Reply:
Ok, 2b813a9